data/tokens.json

```diff
@@ -2494,43 +2494,43 @@
       },
       "type": "typography"
     },
-    "tooltip-lg-padding-horizontal": {
+    "gtmds-tooltip-horizontalPadding-lg": {
       "value": "{lg}",
       "type": "spacing"
     },
-    "tooltip-lg-padding-vertical": {
-      "value": "{lg}-{sm}",
+    "gtmds-tooltip-verticalPadding-lg": {
+      "value": "{md}+{sm}",
       "type": "spacing"
     },
-    "tooltip-bg": {
+    "gtmds-tooltip-backgroundColor": {
       "value": "{gray-10}",
       "type": "color"
     },
-    "tooltip-color": {
+    "gtmds-tooltip-color": {
       "value": "{Absolute.white}",
       "type": "color"
     },
-    "tooltip-border-radius": {
+    "gtmds-tooltip-borderRadius-sm": {
       "value": "{border-radius-sm}",
       "type": "borderRadius"
     },
-    "tooltip-box-shadow": {
+    "gtmds-tooltip-shadow-popover": {
       "value": "{Popover Shadow}",
       "type": "boxShadow"
     },
     "tooltip-font-familiy": {
       "value": "{Paragraph.paragraph-3-font-family}",
       "type": "fontFamilies"
     },
-    "tooltip-sm-padding-horizontal": {
+    "gtmds-tooltip-horizontalPadding-sm": {
       "value": "{sm}*2",
       "type": "spacing"
     },
-    "tooltip-sm-padding-vertical": {
+    "gtmds-tooltip-verticalPadding-sm": {
       "value": "{sm}",
       "type": "spacing"
     },
-    "tooltip-list-padding": {
+    "gtmds-tooltip-padding-list": {
```

Comment:
maybe we are killing this one and moving it to a popover, I will merge anyhow :)